components/03-modules/cart-list-item/mocks/cartListItem.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Review and reduce this mock file, I see there stuff that should be probably hardcoded in the component, like classes.

Also, it will be better to keep only the backend related data in mocks, not whole component configuration.